tests/swagger/test_validation.py

```diff
@@ -113,7 +113,7 @@ def test_validate_headers(self):
         for head in headers:
             self.request.headers = head
             self.assertRaises(ValidationError, unmarshal_request,
-                              self.request, self.resources['Buckets'].get_buckets)
+                              self.request, self.resources['Bucket'].get_buckets)
```

Comment:
Ouch.. too bad we have to make dozens of changes like this. Maybe we should try to find a few minutes in a subsequent PR to extract this expression as an attribute on the test case, something like `self.swagger_resource`?